# sql-alchemy-intro-blogly-part1
http://curric.rithmschool.com/springboard/exercises/flask-blogly/

Fixed previous error. Now get the error: 
======================================================================
FAIL: test_add_user (tests.test-routes.TestRoutes)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/shangita/Desktop/UMASS_Global/Assessment/sql-alchemy/sql-alchemy-intro/sql-alchemy-intro-blogly-part1/tests/test-routes.py", line 52, in test_add_user
    self.assertEqual(response.status_code, 302)
AssertionError: 200 != 302

----------------------------------------------------------------------
Ran 5 tests in 0.175s

FAILED (failures=1)
